rq_dashboard/cli.py

```diff
@@ -106,6 +116,8 @@ def make_flask_app(config, username, password, url_prefix, compatibility_mode=Tr
     '--delete-jobs', default=False, help='Delete jobs instead of cancel')
 @click.option(
     '--debug/--normal', default=False, help='Enter DEBUG mode')
+@click.option(
+    '--verbose-logging', default=False, help='Make Flask logger verbose')
```

Comment:
Could you please also add it to the README?

Reply:
Okay! I've added the new option to the readme.